README.md

```diff
@@ -365,6 +365,10 @@ Share your apps here! Submit a pull request!
 ### Android
 * **Appwrite + Kotlin Android App -> Taleia** - A simple app that generates sets of random word combinations to fight the writer's block. It Allows to log in and save them with Appwrite to be able to retreive the liked combinations later. [Source Code](https://github.com/und1n3/taleia)
 
+### .NET
+
+* **SOS App** - A cross-platform mobile app built with .NET MAUI that allows the user to send an SOS message with their location to a saved phone number in times of distress using an Appwrite Function, Radar, and Twilio. [Source Code](https://github.com/adityaoberai/SOS-MAUI)
```

Comment:
Whoa! Really useful sample of a .NET app! My only concern is with adding an API Key in a client app. Is it needed, especially since 1.0 and above now allows for `any` to be added to functions?

Reply:
Good point 💯
I had initially tried to repeat the flow of how a serverside app would have to try this (because a JWT was not being generated). Let me, however, try to remove the need for an API key entirely.